### ISSUE TITLE
Cocoapods still on 5.1.0

### ISSUE THREAD

**Issue by mrgrauel** [yeahdongcn/UIColor-Hex-Swift#73]
The Cocoapods Specs are still pointing to Version 5.1.0. Is that intended?

https://cocoapods.org/pods/UIColor_Hex_Swift

**Comment by changpk** [comment]
the version number is not synchronized to Cocoapods, I always get an error：
[!] CocoaPods could not find compatible versions for pod "UIColor_Hex_Swift":
  In Podfile:
    UIColor_Hex_Swift (~> 5.1.1)

None of your spec sources contain a spec satisfying the dependency: `UIColor_Hex_Swift (~> 5.1.1)`.

**Comment by yeahdongcn** [comment]
Should be resolved now by aeb4aa99123e740fd365a6ff223fad7d2a9c49a7.

```
-> UIColor_Hex_Swift (5.1.6)
   Convenience method for creating autoreleased color using RGBA hex string.
   pod 'UIColor_Hex_Swift', '~> 5.1.6'
   - Homepage: https://github.com/yeahdongcn/UIColor-Hex-Swift
   - Source:   https://github.com/yeahdongcn/UIColor-Hex-Swift.git
   - Versions: 5.1.6, 5.1.0, 5.0.0, 4.2.0, 4.0.2, 4.0.1, 3.0.2, 3.0.1, 3.0, 2.3, 2.1,
   2.0, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.2, 1.1, 1.0 [trunk repo]
```
